cmd/dump.go

```diff
@@ -232,15 +232,18 @@ var dumpCmd = &cobra.Command{
 
 		oasApiFiles := make([]string, len(oasApisDB))
 		for i, oasApi := range oasApisDB {
-			j, jerr := json.MarshalIndent(oasApi, "", "  ")
+			oasApiC := oasApi
```

Comment:
Why are you assigning it to different variable here?

Reply:
forgot to revert this - i was just verifying something else. i'll correct this one